Assets/LeapMotionModules/Interaction/Examples/TransformHandles/Scripts/RotationHandle.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Leap;
+using Leap.Unity.UI.Interaction;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Examples.TransformHandles {
+
+  public class RotationHandle : InteractionTransformHandle {
+
+    protected override void Start() {
+      base.Start();
+      
+      // Parent class stores the InteractionBehaviour component in _intObj.
+      _intObj.OnGraspBegin += onGraspBegin;
```

Comment:
Same thing here with the OnEnable/OnDisable used for subscription/unsubscription